### DISCUSSION TITLE
挂载阿里云盘open提示failed init storage: failed to refresh token: The input parameter refresh_token is not valid.

### DISCUSSION THREAD

**Discussion by likepoems** [AlistGo/alist#3607] [Q&A]
挂载阿里云盘open提示failed init storage: failed to refresh token: The input parameter refresh_token is not valid.
按照https://alist.nn.ci/tool/aliyundrive/request.html获取的token保存报错

**Comment by SeanYang24** [comment]
我也有同样的问题

**Comment by kidqian** [comment]
一样，等解决。

**Comment by likepoems** [comment]
升到最新版重新添加就解决了

**Reply by SeanYang24** [reply]
真的哦！！解决了

**Comment by szh811218** [comment]
我升级后重新添加了还是同样的报错

**Reply by szh811218** [reply]
我查了一下版本，我本地的版本是3.11.0，是不是需要升级3.12?

**Reply by likepoems** [reply]
> 我查了一下版本，我本地的版本是3.11.0，是不是需要升级3.12?

需要升级到最新版，之前的不可用。阿里云盘需要重新添加，分为open/share，文档中有描述。

**Reply by szh811218** [reply]
升级到3.12已经解决，谢谢

**Comment by Arc-2023** [comment]
最新的3.12.2那个阿里云盘接口好像还有这个问题

**Comment by 0xYiliu** [comment]
3.13.2也不行，从文档中扫二维码获取的token

**Comment by shakebaby2023** [comment]
还是不行，肯定有问题

**Comment by szh811218** [comment]
目前使用3.13可以正常用

**Reply by dexle** [reply]
我升级到3.13.2 还是不行 第一次升级到3.12 可以用一两天 就不行 继续升级 就一直不成功了~~

**Reply by szh811218** [reply]
我目前也是3.13.2，是可以用的，你试试重新挂载一下

**Comment by likepoems** [comment]
我的是3.12.2，可以正常使用
https://pan.likepoems.com

**Comment by AoThen** [comment]
检查一下,refresh_token 开头是不是有空格,删除一下.....

**Comment by kenyon01** [comment]
我出现这个错误是因为我用的是阿里云盘open的刷新令牌，但是挂载的阿里云盘而不是阿里云盘open。后面我更新alist到最新版，同时改正成了阿里云盘open，解决了这个问题。

**Reply by eons2long** [reply]
谢谢 原来是这样，我也是因为这个出错的

**Comment by Yihangmarss** [comment]
解决了
1. https://alist.nn.ci/tool/aliyundrive/ 从这里获取时，选“ https://api.xhofe.top (cn) ”
2. 在 http://127.0.0.1:5244 的网站 `Oauth令牌链接` 输入“ https://api.xhofe.top/alist/ali_open/token”

**Reply by 591210009** [reply]
按照你的方法 解决了 谢谢

**Reply by SunSeaLucky** [reply]
<img width="1187" height="387" alt="image" src="https://github.com/user-attachments/assets/2b3fcc26-542b-452f-8cdd-2fced0fb80a7" />
这里似乎只有 cf 结尾的可选。且按照 @Yihangmarss 的操作指引，提示：

> Failed init storage: failed to refresh token: refresh token is empty, resp: 404 page not found
